Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -351,7 +351,8 @@ bool AXObjectCache::modalElementHasAccessibleContent(Element& element)
                 // When using ATSPI, an accessibility object with 'StaticText' role is ignored.
                 // Its content is exposed by its parent.
                 // Treat such elements as having accessible content.
-                if (axObject->roleValue() == AccessibilityRole::StaticText)
+                // FIXME: This may not be sufficient for visibility:hidden or inert (https://bugs.webkit.org/show_bug.cgi?id=280914)
```

Comment:
@twilco here is the updated comment + condition (with new bug https://bugs.webkit.org/show_bug.cgi?id=280914 filed)